src/components/Authentication/PasswordInput.js

```diff
@@ -1,45 +1,53 @@
-import React, {useCallback, useState} from "react";
-import {makeStyles} from "@material-ui/core/styles";
-
-import FormControl from "@material-ui/core/FormControl";
+import React, { useCallback, useState } from "react";
 import IconButton from "@material-ui/core/IconButton";
-import Input from "@material-ui/core/Input";
 import InputAdornment from "@material-ui/core/InputAdornment";
-import InputLabel from "@material-ui/core/InputLabel";
 import Visibility from "@material-ui/icons/Visibility";
 import VisibilityOff from "@material-ui/icons/VisibilityOff";
+import TextField from "@material-ui/core/TextField";
+import VpnKeyOutlinedIcon from '@material-ui/icons/VpnKeyOutlined';
+import "./LoginPage.scss";
 
-const useStyles = makeStyles({
-    "root": {
-        "width": "100%",
-    },
-});
-
-const PasswordInput = ({label = "Password", ...props}) => {
+const PasswordInput = ({ label = "Password", ...props }) => {
     const [showPassword, setShowPassword] = useState(false);
+    const [password, setPassword] = useState(null);
 
     const toggleVisibility = useCallback(() => {
         setShowPassword((show) => !show);
+
+    }, []);
```

Comment:
add dependencies `setShowPassword` and `show` in your empty array

Reply:
Not necessary as is currently written